.github/workflows/test-latest-release.yml

```diff
@@ -0,0 +1,168 @@
+name: Test latest release
+
+on:
+  workflow_call:
+  # So we can trigger manually if needed
+  workflow_dispatch:
+  push:
+    branches:
+      - release-*
+    tags:
+      - 'v*'
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
```

Comment:
I suspect that this may cause us problems by suppressing activation for a commit that only changes docs. It is not uncommon for the last commit in a release to be one that only changes docs (e.g., fixing self-reference(s)).

Reply:
You're right.  And my bad for letting this one slip in.